protoc-gen-go/golden_test.go

```diff
@@ -123,3 +111,181 @@ func TestGolden(t *testing.T) {
 }
 
 var fdescRE = regexp.MustCompile(`(?ms)^var fileDescriptor.*}`)
+
+func TestParameters(t *testing.T) {
+	for _, test := range []struct {
+		parameters  string
+		wantFiles   map[string]bool
+		wantImports map[string]bool
```

Comment:
nit: rename with A in the name somewhere? maybe wantImportsA?

Reply:
done